spec/unit/scrum/sprint_cleaner_spec.rb

```diff
@@ -8,8 +8,10 @@
   end
 
   it 'moves remaining cards to target board', vcr: 'sprint_cleanup', vcr_record: false do
-    expect(STDOUT).to receive(:puts).exactly(13).times
-    expect(subject.cleanup('7Zar7bNm', '72tOJsGS')).to be
+    allow(Trello::Label).to receive(:create)
+    allow_any_instance_of(Trello::Card).to receive(:add_label).and_return(true)
+    expect(STDOUT).to receive(:puts).exactly(25).times
```

Comment:
@Ana06 I increased the check from the previous value 13 to 25. I added a puts statement in [add_in_last_sprint_label](https://github.com/openSUSE/trollolo/pull/156/files#diff-eecdf5816db86046bb09563f62a60f0cR44) That amounts for the extra 12. Also now I'm using  [`add_label`](https://github.com/jeremytregunna/ruby-trello/blob/master/lib/trello/card.rb#L402) method of the library to add our new label, whose call I've stubbed in the tests.

Reply:
I don't really like to modify the code to make a test passing...

Reply:
and the comment is not really testing that the card label is added...